tests/conftest.py

```diff
@@ -37,15 +38,16 @@
 import pytest
 from gluon.contrib.webclient import WebClient
 import gluon.shell
-from py_w3c.validators.html.validator import HTMLValidator
 import six
 from six.moves.urllib.error import HTTPError, URLError
 from six.moves.urllib.request import urlopen
+from html5validator.validator import Validator
 
 # Local imports
 # -------------
 from .utils import COVER_DIRS, DictToObject
 
+W3_VALIDATE = True
```

Comment:
I'd recommend adding a brief comment to explain what this does. Another handy option would be to add a command-line flag to disable it.

Reply:
Yep.